framework/include/ble_manager/ble_client.h

```diff
@@ -0,0 +1,293 @@
+
+/****************************************************************************
+ * Name: ble_client_delete_bond
+ *
+ * Description:
+ *   Delete the bonded device.
+ *
+ * Input Parameters:
+ *   addr      - This value should include bd_addr and addr_type.
+ *
+ * Returned Value
+ *   Zero (BLE_RESULT_SUCCESS) is returned on success; a positive value is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ble_result ble_client_delete_bond(ble_client_bd_addr* addr);
+
+/****************************************************************************
+ * Name: ble_client_delete_bond_all
+ *
+ * Description:
+ *   Delete all of the bonded devicse.
+ *
+ * Returned Value
+ *   Zero (BLE_RESULT_SUCCESS) is returned on success; a positive value is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ble_result ble_client_delete_bond_all(void);
+
+/****************************************************************************
+ * Name: ble_client_disconnect
+ *
+ * Description:
+ *   Disconnect with connected handle ID.
+ *
+ * Input Parameters:
+ *   conn_handle  - The handle ID of connected device.
+ * 
+ * Returned Value
+ *   Zero (BLE_RESULT_SUCCESS) is returned on success; a positive value is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ble_result ble_client_disconnect(ble_conn_handle conn_handle);
+
+/****************************************************************************
+ * Name: ble_client_disconnect_all
+ *
+ * Description:
+ *   Disconnect all of the connected devicse.
+ *
+ * Returned Value
+ *   Zero (BLE_RESULT_SUCCESS) is returned on success; a positive value is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ble_result ble_client_disconnect_all(void);
+
+/****************************************************************************
+ * Name: ble_client_operation_enable_notification
+ *
+ * Description:
+ *   Enable the notification comming from the Server(Peripheral).
+ * 
+ * Input Parameters:
+ *   handle  - This includes the handle ID and the attribute ID.
+ * 
+ * Returned Value
+ *   Zero (BLE_RESULT_SUCCESS) is returned on success; a positive value is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ble_result ble_client_operation_enable_notification(ble_client_operation_handle* handle);
+
+/****************************************************************************
+ * Name: ble_client_operation_read
+ *       ble_client_operation_write
+ *       ble_client_operation_write_no_response
+ *
+ * Description:
+ *   I/O operation with Server(Peripheral).
+ * 
+ * Input Parameters:
+ *   handle  - This includes the handle ID and the attribute ID.
+ *   dat     - I/O buffer which trasfers data.
```

Comment:
How about changing to `in_data/out_data` or `data`? Because `dat` is not easy to understand.

Reply:
I fixed it